andela/codepirates-ah-frontend
# #167313368 User should be able to clap for and dislike an article

#### What does this PR do?
 User should be able to clap for and dislike a comment.
#### Description of Task to be completed?
I want to be able to clap or dislike articles, both authored by me and those not authored by me.
#### How should this be manually tested?
Visit single article view.
User should be able to dislike or clap for that particular article
#### What are the relevant pivotal tracker stories?
[#167313368](https://www.pivotaltracker.com/story/show/167313368)
<img width="1440" alt="Screen Shot 2019-11-03 at 20 17 49" src="https://user-images.githubusercontent.com/31512000/68089965-177ad500-fe77-11e9-94fd-6ede93d9dc15.png">
<img width="1440" alt="Screen Shot 2019-11-03 at 20 16 58" src="https://user-images.githubusercontent.com/31512000/68089969-1ea1e300-fe77-11e9-96ea-d8f271bac313.png">
